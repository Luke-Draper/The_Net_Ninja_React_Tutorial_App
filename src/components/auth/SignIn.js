import React, { Component } from "react";

class SignIn extends Component {
	render() {
		return (
			<div className="sign-in">
				<h3>SignIn</h3>
			</div>
		);
	}
}

export default SignIn;
