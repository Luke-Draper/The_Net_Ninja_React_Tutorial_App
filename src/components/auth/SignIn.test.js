import React from "react";
import SignIn from "./SignIn";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("SignIn renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<SignIn />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
