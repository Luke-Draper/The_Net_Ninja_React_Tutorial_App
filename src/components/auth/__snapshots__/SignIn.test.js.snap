// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn renders without crashing 1`] = `
<div
  className="sign-in"
>
  <h3>
    SignIn
  </h3>
</div>
`;
